tenprintcover.py

```diff
@@ -498,25 +498,43 @@ def drawShape(c, x, y, s):
         else:
             assert not "Implement."
 
-
+    def scale_font(text, font_name, font_properties):
+        # very dumb heuristic
+        (font_size, font_slant, font_weight) = font_properties
+        w = len(text) * font_size
+        print 100 * w
+        if w > cover_width * 7:
+            return  (font_size * 0.8, font_slant, font_weight)
+        else:
+            return font_properties
+        
     # Allocate fonts for the title and the author, and draw the text.
     def drawText():
         fill = Image.colorRGB(50, 50, 50)
 
         title_font_size = cover_width * 0.08
+        subtitle_font_size = cover_width * 0.05
         title_font_properties = (title_font_size, cairo.FONT_SLANT_NORMAL, cairo.FONT_WEIGHT_BOLD)
-        title_font = cover_image.font("Avenir Next", title_font_properties)
+        subtitle_font_properties = (subtitle_font_size, cairo.FONT_SLANT_NORMAL,
+                                     cairo.FONT_WEIGHT_NORMAL)
+        title_font_properties = scale_font(title, "Arial Unicode MS", title_font_properties)
```

Comment:
Perhaps [Noto Sans](https://fonts.google.com/specimen/Noto+Sans) would be a safe option, considering it covers the Unicode pages completely.